README.md

```diff
@@ -49,6 +56,21 @@ make
 make test
 ```
 
+Optionally, you can install [DeepMind Lab](https://deepmind.com/blog/open-sourcing-deepmind-lab) and [Project Malmo](https://www.microsoft.com/en-us/research/project/project-malmo).
+
+To use DeepMind Lab, please follow the instructions on [https://github.com/deepmind/lab](https://github.com/deepmind/lab) and run the cmake command
+```
+cmake -DWITH_DEEPMIND_LAB=ON ..
+```
+which will automatically download and build DeepMind Lab.
+
+To use Malmo, please first follow the instructions on [https://github.com/Microsoft/malmo](https://github.com/Microsoft/malmo) to install Malmo, and then set the environment variable `MALMO_ROOT` to the location where Malmo is installed
+```
+cd <xworld_path>
+./export_vars.sh MALMO_ROOT /path/to/MALMO
```

Comment:
The correct way to do this should be cmake -DMALMO_ROOT=/path/to/MALMO. The CMake file will automatically add MALMO_ROOT to bashrc